azkaban-db/src/main/sql/create.execution_flows.sql

```diff
@@ -15,6 +15,7 @@ CREATE TABLE execution_flows (
   use_executor INT                  DEFAULT NULL,
   flow_priority TINYINT    NOT NULL DEFAULT 5,
   execution_source VARCHAR(32)        DEFAULT NULL,
+  dispatch_method TINYINT        DEFAULT 1,
```

Comment:
Can you please add index for this field?

Reply:
done

Reply:
Upgrade sql was not added for this change, but it should be (under https://github.com/azkaban/azkaban/tree/master/azkaban-db/src/main/sql).

Same for the added CREATE INDEX below.